src/aspire/utils/misc.py

```diff
@@ -121,57 +120,65 @@ def gaussian_1d(size, mu=0, sigma=1, peak=1, dtype=np.float64):
 
     p = (g["x"] - mu) ** 2 / (2 * sigma**2)
 
-    return (peak * np.exp(-p)).astype(dtype, copy=False)
+    return np.exp(-p).astype(dtype, copy=False)
 
 
-def gaussian_2d(size, x0=0, y0=0, sigma_x=1, sigma_y=1, peak=1, dtype=np.float64):
+def gaussian_2d(size, mu=(0, 0), sigma=(1, 1), dtype=np.float64):
     """
     Returns a 2d Gaussian in a square 2d numpy array.
 
-    Default is a centered disc of spread=peak=1.
+    Default is a centered disc of spread=1.
 
     :param size: The height and width of returned array (pixels)
-    :param x0: x coordinate of center (pixels)
-    :param y0: y coordinate of center (pixels)
-    :param sigma_x: spread in x direction
-    :param sigma_y: spread in y direction
-    :param peak: peak height at center
+    :param mu: A 2-tuple indicating the center of the Gaussian
+    :param sigma: A 2-tuple (or scalar) of spreads corresponding to mu
```

Comment:
“corresponding”?

Reply:
Reworded to "standard deviation in the x and y directions". Not sure if we should go with "standard deviation" since this is not a probability distribution, but I think it is the standard language when speaking of Gaussian functions. Let me know if you have another suggestion.

Reply:
I think that's fine.